modules/odc-interacting-with-applications-and-components.adoc

```diff
@@ -5,21 +5,21 @@
 
-The *Topology* view in the *Developer* perspective of the web console provides the following options to interact with the application and the components:
+By default, the *Topology view* renders in the *Developer* perspective of the web console provides the following options to interact with the application and the components:
 
-* Click *Open URL* (image:odc_open_url.png[title="Application Link"]) to see your application exposed by the route on a public URL.
-* Click *Edit Source code* to access your source code and modify it.
+* Use the *List View* icon to see a list of all your applications and use the *Topology View* icon to switch back to the *Topology* view.
+** Click *Open URL* (image:odc_open_url.png[title="Application Link"]) to see your application exposed by the route on a public URL.
+** Click *Edit Source code* to access your source code and modify it.
 +
 [NOTE]
 ====
 This feature is available only when you create applications using the *From Git*, *From Catalog*, and the *From Dockerfile* options.
 ====
 +
-* Hover your cursor over the lower left icon on the Pod to see the name of the latest build and its status. The status of the application build is indicated as *New* (image:odc_build_new.png[title="New Build"]), *Pending* (image:odc_build_pending.png[title="Pending Build"]), *Running* (image:odc_build_running.png[title="Running Build"]), *Completed* (image:odc_build_completed.png[title="Completed Build"]), *Failed* (image:odc_build_failed.png[title="Failed Build"]), and *Canceled* (image:odc_build_canceled.png[title="Canceled Build"]).
-* Use the *Shortcuts* menu listed on the upper-right of the screen to navigate components in the *Topology* view.
-* Use the *List View* icon to see a list of all your applications and use the *Topology View* icon to switch back to the *Topology* view.
-* Use the *Find by name* field to select the components with component names that match the query. Search results may appear outside of the visible area; click *Fit to Screen* from the lower-left toolbar to resize the *Topology* view to show all components.
-* Use the *Display Options* drop-down list to configure the *Topology* view of the various application groupings. The options are available depending on the types of components deployed in the project:
+** Hover your cursor over the lower left icon on the Pod to see the name of the latest build and its status. The status of the application build is indicated as *New* (image:odc_build_new.png[title="New Build"]), *Pending* (image:odc_build_pending.png[title="Pending Build"]), *Running* (image:odc_build_running.png[title="Running Build"]), *Completed* (image:odc_build_completed.png[title="Completed Build"]), *Failed* (image:odc_build_failed.png[title="Failed Build"]), and *Canceled* (image:odc_build_canceled.png[title="Canceled Build"]).
+* Use the *View Shortcuts* menu listed on the upper-right of the screen to navigate components in the *Topology view*.
```

Comment:
let's move the bullet points from here to the start before we list the topology view bullet points.